prow/cmd/mkpod/local-kind-config.yaml

```diff
@@ -0,0 +1,7 @@
+# Example kind config for getting output on the machine running kind instead of the contain 'node'.
```

Comment:
`contain 'node'`?

Reply:
`container 'node'` or `'node' container`. Spelling is hard.

Reply:
outstanding?